### PR TITLE
Add genre list

### PR DESCRIPTION
**Pull request by auhc99**

### CODE REVIEW

File: src/main/java/seedu/anilist/model/genre/Genre.java [hunk 1]
```diff
@@ -30,7 +33,8 @@ public Genre(String genreName) {
      * Returns true if a given string is a valid genre name.
      */
     public static boolean isValidGenreName(String test) {
-        return test.matches(VALIDATION_REGEX);
+        requireNonNull(test);
+        return GenreList.contains(test.toLowerCase(Locale.ROOT));
```
Comment (clyveycui) [inline src/main/java/seedu/anilist/model/genre/Genre.java:37]: I think Locale.ROOT May not be too necessary

File: src/main/java/seedu/anilist/logic/commands/Action.java [hunk 2]
```diff
@@ -17,6 +17,8 @@
      */
     public static boolean isValidAction(String test) {
         switch (test) {
+        case "a":
+        case "d":
         case "add" :
         case "delete" :
             return true;
```
Comment (clyveycui) [inline src/main/java/seedu/anilist/logic/commands/Action.java:24]: Is this function updated for list?

File: src/main/java/seedu/anilist/logic/commands/Action.java [hunk 3]
```diff
@@ -32,8 +34,10 @@ public static Action actionFromString(String actionString) {
         assert actionString != null;
 
         switch (actionString) {
+        case "a":
         case "add" :
             return Action.ADD;
+        case "d":
         case "delete" :
             return Action.DELETE;
         default:
```
Comment (clyveycui) [inline src/main/java/seedu/anilist/logic/commands/Action.java:43]: same here

File: src/test/java/seedu/anilist/model/genre/GenreListTest.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            assertNotEquals(wrongString, actualString);
+        }
+    }
+
+    /**
+     * contains should return true if given
+     * string is inside GenreList, false otherwise
+     */
+    @Test
+    public void contains() {
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/model/genre/GenreListTest.java:46]: check null input

**Review by clyveycui — COMMENTED** [review]
Just a few issues, please take a look.

File: src/main/java/seedu/anilist/logic/commands/Action.java (hunk 2 above)
Comment (auhc99) [reply src/main/java/seedu/anilist/logic/commands/Action.java:24]: Havnt add the list action word yet, thinking of doing it in another branch

File: src/main/java/seedu/anilist/logic/commands/Action.java (hunk 3 above)
Comment (auhc99) [reply src/main/java/seedu/anilist/logic/commands/Action.java:43]: same

File: src/test/java/seedu/anilist/model/genre/GenreListTest.java (hunk 4 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/genre/GenreListTest.java:47]: added

**Pull request merged**